custom_components/powercalc/data/cree/B00TN589ZG/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Sonoff S31 running Tasmota",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.3:docker"
+    },
+    "name": "Cree Connected A-19 60W Equivalent",
```

I suggest to remove Cree from the model name as it is the brand, which is already displayed separately in the model listing.

```suggestion
    "name": "Connected A-19 60W Equivalent",
```

---

Done. Thank you.